docs/release-notes/8.8.asciidoc

```diff
@@ -18,14 +18,15 @@ To view a detailed summary of new features and enhancements we've documented thi
 
 //tag::breaking-changes[]
 // NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+// THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE FULL URLS WITH NO ATTRIBUTES
 :pull: https://github.com/elastic/kibana/pull/
 * The privileges for attaching alerts to cases have changed. Now, you need at least `Read` privileges for Security and `All` privileges for Cases ({pull}147985[#147985]).
 * Adds conditional actions to the rules API. In {elastic-sec} 8.7 and earlier, action frequencies were set on a rule level by defining the `throttle` field. In 8.8 and later, action frequencies are set at the action level, and the `throttle` field is replaced by the `frequency` and `alert_filters` fields. The following APIs are affected:
-** <<rules-api-get,Get rule>>
-** <<rules-api-find,Find rules>>
-** <<rules-api-create,Create rule>>
-** <<rules-api-update,Update rule>>
-** <<bulk-actions-rules-api,Bulk rule actions>>
+** https://www.elastic.co/guide/en/security/8.8/rules-api-get.html[Get rule]
+** https://www.elastic.co/guide/en/security/current/rules-api-find.html[Find rules]
+** https://www.elastic.co/guide/en/security/current/rules-api-create.html[Create rule]
+** https://www.elastic.co/guide/en/security/current/rules-api-update.html[Update rule]
+** https://www.elastic.co/guide/en/security/current/bulk-actions-rules-api.html[Bulk rule actions]
```

Comment:
I'll need to merge https://github.com/elastic/security-docs/pull/3320 before this link can resolve. 

```suggestion
** https://www.elastic.co/guide/en/security/8.8/bulk-actions-rules-api.html#optional-actions-fields-bulk-update[Bulk rule actions]
```